atom/browser/api/atom_api_session.cc

```diff
@@ -419,9 +419,9 @@ v8::Local<v8::Promise> Session::ResolveProxy(mate::Arguments* args) {
   args->GetNext(&url);
 
   browser_context_->GetResolveProxyHelper()->ResolveProxy(
-      url,
-      base::Bind(util::CopyablePromise::ResolveCopyablePromise<std::string>,
-                 util::CopyablePromise(promise)));
+      url, base::BindRepeating(
```

Comment:
should `ResolveProxyCallback` be `ResolveProxyOnceCallback`?

Reply:
@nornagon we don't enforce this with any degree of standardization across the codebase, but i'm happy to change it if you think it'd be clearer :)

Reply:
Well, given that it looks like the resolution only happens once, I think `BindOnce` is warranted.

Reply:
done!